awx/ui/src/screens/Instances/Shared/RemoveInstanceButton.js

```diff
@@ -0,0 +1,206 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { t, Plural } from '@lingui/macro';
+import { KebabifiedContext } from 'contexts/Kebabified';
+import {
+  getRelatedResourceDeleteCounts,
+  relatedResourceDeleteRequests,
+} from 'util/getRelatedResourceDeleteDetails';
+import {
+  Button,
+  DropdownItem,
+  Tooltip,
+  Alert,
+  Badge,
+} from '@patternfly/react-core';
+import AlertModal from 'components/AlertModal';
+import styled from 'styled-components';
+
+const WarningMessage = styled(Alert)`
+  margin-top: 10px;
+`;
+
+const Label = styled.span`
+  && {
+    margin-right: 10px;
+  }
+`;
+
+function RemoveInstanceButton({
+  itemsToRemove,
+  onRemove,
+  clearSelected,
+  isK8s,
+}) {
+  const { isKebabified, onKebabModalChange } = useContext(KebabifiedContext);
```

Comment:
I haven't been able to get this to build locally, however I thought we had discussed with @trahman73 to not use kebabs this release and stick with our established design pattern of the delete button like on other lists?

Reply:
This kebab is used in Advanced search mode.  It is not on the list items.